libs/ui/src/lib/main-menu/data/index.ts

```diff
@@ -41,16 +41,22 @@ const getMainMenuData = (dictionary: Dictionary, baseUri: string) => ({
 	],
 	help: [
 		{
-			title: dictionary.mainMenu.help.links.getStarted,
-			href: `${baseUri}/veiledning`
+			title: dictionary.mainMenu.help.links.findingData,
+			href: `${baseUri}/getting-started/finding-data`
+		},
+		{
+			title: dictionary.mainMenu.help.links.sharingData,
+			href: `${baseUri}/getting-started/publishing-intro`
 		},
 		{
```

Comment:
Legge til en entry "Hvordan dele data" som peker til Nasjonal verktøykasse for deling av data: https://www.digdir.no/datadeling/dele-data-med-andre/2252

![Screenshot 2024-08-14 at 10 18 40](https://github.com/user-attachments/assets/90a3c8d8-2d8d-411a-a907-7f7fb4f15b11)